src/status_im/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -249,6 +249,16 @@
             user-props                {:full-name to-address
                                        :address   (utils/get-shortened-address
                                                    to-address)}]
+        ;; In token send flow we already have transaction built when
+        ;; we reach confirmation screen. But in send collectible flow
+        ;; routes request happens at the same time with navigation to
+        ;; confirmation screen. So we need to build the transaction as soon
+        ;; as route is available.
+        (rn/use-effect
+         (fn []
+           (when (and (= transaction-type :tx/collectible-erc-1155) first-route)
```

Comment:
Quick question: why not `:tx/collectible-erc-721`  is added?

Reply:
@smohamedjavid, great catch, thanks you!

Reply:
No worries 🙌 

Sorry for the last minute comment. For collectible tx type check there is a util method. Please use it 🙏 

https://github.com/status-im/status-mobile/blob/2be78041e2b5e0728a603155aaeeeacf427efff2/src/status_im/contexts/wallet/send/utils.cljs#L292-L294

Reply:
@smohamedjavid, ideal! thank you :)